src/api/goods.js

```diff
@@ -6,6 +6,20 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => (
+  getAll()
+    .then(products => (
+      products.sort((prevProduct, currentProduct) => (
+        prevProduct.name.localeCompare(currentProduct.name)
+      ))
+    ))
+    .then(products => products.splice(0, 5))
```

Comment:
Навіщо тобі видаляти перші 5 елементів ?
```suggestion
    .then(products => products.slice(0, 5))
```